test/jdk/javax/swing/JOptionPane/8081019/bug8081019.java

```diff
@@ -31,7 +31,6 @@
  * @key headful
```

Comment:
javadoc style /** at the beginning

Reply:
Not specific to JEP 486, this should be done as part of a different issue.

Reply:
agreed. No need to touch.